### DISCUSSION TITLE
When will Amazfit GTR Mini receive Zepp OS 2.1

### DISCUSSION THREAD

**Discussion by Humorist2601** [zepp-health/zeppos-forum#244] [Q&A] [answered]
Hello, I wanted to know when will you update GTR Mini to Zepp OS 2.1 as it is the only device that is currently stuck on version 2.0. I found an answer that it will be update on [June](https://github.com/orgs/zepp-health/discussions/202#discussioncomment-6293944) but no other discussion since then

**Comment by MyoungXUE** [comment, marked as answer]
@TheHawaiianKoala 

Hello, thanks for your interest in Zepp OS, the colleagues involved have been working hard on the GTR mini upgrade for some time now!

In order to ensure the stability of the system and the best performance, we usually have a long upgrade cycle, but as far as I know, it's in the final stage, so please be patient for a while!

**Reply by Humorist2601** [reply]
If you want, I can beta test the build for you guys.
